ostd/src/mm/page_table/node/entry.rs

```diff
@@ -24,10 +25,10 @@ pub(in crate::mm) struct Entry<'a, E: PageTableEntryTrait, C: PagingConstsTrait>
     /// The index of the entry in the node.
     idx: usize,
     /// The node that contains the entry.
-    node: &'a mut PageTableNode<E, C>,
+    node: &'guard mut PageTableGuard<'pt, E, C>,
```

Comment:
For consistency, let's define a rule for naming the lifetime. The RCU grace period seems to be quite important, so it deserves a named lifetime (i.e. `'rcu`). I'm not sure if the `'guard` lifetime makes sense; if not, let's just use `'a`.

Note that by this rule I mean that it should apply to all related files, not just this one. Naming all lifetimes under the `page_table` directory consistently should be beneficial.